### ISSUE TITLE
Create Advisor chat interface

### ISSUE THREAD

**Issue by FoRVaiS** [Vero-Ventures/insurance-calculator#91]
## Description
The chat interface allows the user to interact with the Advisor accompanied by the user's profile information.

## Technical
The chat interface will not retain chat history nor will it have multiple chat sessions. There will be a button to reset the chat to remove all previous messages from context.

The interface will have a max width of ~100 characters to maintain readability.

Prompt requests sent to the advisor will hang until it is completed, timed out (or errored), or aborted. Changes in the Advisor service can be made to return an id that can be used to poll a "get response" endpoint but that is out of scope for this task.

### States
The chat interface has the following states:
 - READY: Advisor is online and ready for input
 - WAIT: Waiting on Advisor response
 - OFFLINE: Advisor is offline or unable to be reached

#### `READY`
The advisor is online and ready for input. Chat bar is unlocked.

#### `WAIT`
The advisor has received a prompt and is currently generating an answer. The chat submit button must be locked out until the Advisor has timed out or returned a response.

Time out can be possibly identified by a 500 error?

#### `OFFLINE` 
Advisor is offline, a message should be displayed notifying the user the Advisor service is offline, and lock the chat bar.

### `POST /api/v1/query`
Request Body:
```json
{
    "prompt": string
}
```
Response:
```json
{
    "data": string
}
```

### `GET /healthz`
If the advisor service is online, the 'advisor' property returns True; else False.
```json
{
    "advisor": boolean
}
```
